src/meta/api/src/schema_api_keys.rs

```diff
@@ -294,3 +296,48 @@ impl KVApiKey for CountTablesKey {
         Ok(CountTablesKey { tenant })
     }
 }
+
+// __fd_table_stage_file/tenant/db_id/table_id/file_name -> TableStageFileInfo
+impl KVApiKey for TableStageFileNameIdent {
+    const PREFIX: &'static str = PREFIX_TABLE_STAGE_FILE;
+
+    fn to_key(&self) -> String {
+        format!(
+            "{}/{}/{}/{}/{}",
+            Self::PREFIX,
+            self.tenant,
+            self.db_id,
+            self.table_id,
+            self.file,
+        )
+    }
+
+    fn from_key(s: &str) -> Result<Self, KVApiKeyError> {
+        let mut elts = s.split('/');
```

Comment:
Maybe buggy. Filename could be `path/to/file`.

Reply:
add `KVApiKeyError::AtleastSegments` to handle these case, and add some test in `schema_api_keys.rs`.